test/images/sample-device-plugin/sampledeviceplugin.go

```diff
@@ -84,14 +91,54 @@ func main() {
 		klog.Errorf("Empty pluginSocksDir")
 		return
 	}
+
+	if autoRegisterString := os.Getenv("AUTO_REGISTER"); autoRegisterString != "" {
+		autoRegister, err = strconv.ParseBool(autoRegisterString)
+		if err != nil {
+			klog.Errorf("invalid boolean value specified")
+			panic(err)
+		}
+
+	}
+
 	socketPath := pluginSocksDir + "/dp." + fmt.Sprintf("%d", time.Now().Unix())
+	triggerPath := pluginSocksDir + "/registered"
```

Comment:
This line can be moved down to the `if !autoRegister` block as it's used only there.

Reply:
Ack, Thanks!